packages/dd-trace/src/opentracing/span_context.js

```diff
@@ -11,6 +11,7 @@ class DatadogSpanContext {
 
     this._traceId = props.traceId
     this._spanId = props.spanId
+    this._isRemote = props.isRemote || null
```

Comment:
We should try to avoid polymorphic types wherever possible. Keep it always bool and it'll be faster as it won't add extra byte code to handle the different type variants.

```suggestion
    this._isRemote = props.isRemote || false
```